runtime/oti/j9nonbuilder.h

```diff
@@ -1367,6 +1367,7 @@ typedef struct J9SharedCacheAPI {
 	I_32 minJIT;
 	I_32 maxJIT;
 	U_8 sharedCacheEnabled;
+	U_8 inContainer;
```

Comment:
This should have a comment that its only set under certain conditions, when -Xshareclasses is not specified, and J9_SHARED_CACHE_DEFAULT_BOOT_SHARING(vm). Or a name that makes it more obvious.

Reply:
Comment added